### PR TITLE
🔀🔖 4.1.3

### PR DESCRIPTION
**Pull request by jxn-30**
* ✨ you can now enable / disable sending telemetry stats in settings
* ✨ you can now colorize the whole navbar via settings (general settings)
* 🚚 we moved the BG setting for LSSM Icon to general settings
* 🌐 updated a cs_CZ loca for telemetry
* 🌐 fixed a nl_NL typo
* 📝 added english files for pl_PL docs

### CODE REVIEW

File: docs/pl_PL/apps.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: ℹ️ General
+lang: en_US
```
Comment (Suchty112) [inline docs/pl_PL/apps.md:3]: hier ist noch en_US statt pl_PL 🙈

File: docs/pl_PL/bugs.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Known Bugs 🐛
+lang: en_US
```
Comment (Suchty112) [inline docs/pl_PL/bugs.md:3]: hier auch

**Review by Suchty112 — COMMENTED** [review]
joa

**Review by Suchty112 — APPROVED** [review]

**Comment by KokosnussDEV** [comment]
@Ron31 is mentioned in the de version as "Praktikant". You wanna add it to the other versions too or just have it only in the German version?

**Comment by jxn-30** [comment]
> @Ron31 is mentioned in the de version as "Praktikant". You wanna add it to the other versions too or just have it only in the German version?

DE only as it is an easter egg :)

**Review by Ron31 — APPROVED** [review]

**Pull request merged**